sdk/eventhub/Azure.Messaging.EventHubs.Processor/perf/Azure.Messaging.EventHubs.Processor.Perf.csproj

```diff
@@ -6,6 +6,10 @@
     <LangVersion>latest</LangVersion>
     <IsTestSupportProject>true</IsTestSupportProject>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
```suggestion

  <ItemGroup>
```

Reply:
should this be removed?

Reply:
Nope; the omission is intentional.   I just forgot to make the spacing pretty.  😄